Wikipedia/Localizations/azb.lproj/Localizable.strings

```diff
@@ -46,7 +46,7 @@
 "search-recent-clear-cancel" = "وازگئچ";
 "search-recent-clear-delete-all" = "هامیسین سیل";
 "main-menu-title" = "آرتیق";
-"main-menu-language-title" = "ویکی‌پئدیادا 1$ آختار";
+"main-menu-language-title" = "ویکی‌پئدیادا $1 آختار";
```

Comment:
note, this looks wrong in GitHub web UI and Xcode. you need to `cat` the file or view in Terminal editor to see the actual string data.